jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/config/PropertiesCustomizer.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper.config;
+
+import static io.opentelemetry.contrib.jmxscraper.config.JmxScraperConfig.JMX_INTERVAL_LEGACY;
+import static io.opentelemetry.contrib.jmxscraper.config.JmxScraperConfig.METRIC_EXPORT_INTERVAL;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+
+/** Customizer of default SDK configuration and provider of effective scraper config */
+public class PropertiesCustomizer implements Function<ConfigProperties, Map<String, String>> {
+
+  private static final Logger logger = Logger.getLogger(PropertiesCustomizer.class.getName());
+
+  private static final String METRICS_EXPORTER = "otel.metrics.exporter";
+
+  @Nullable private JmxScraperConfig scraperConfig;
+
+  @Override
+  public Map<String, String> apply(ConfigProperties config) {
+    Map<String, String> result = new HashMap<>();
```

Comment:
[for reviewer] the returned map contains the configuration items that will be explicitly set/overriden so that they can be later read by the SDK initialization. In addition to that we use this to build an instance of `JmxScraperConfig` from the provided `ConfigProperties` that will later be used to configure the jmx scraper.